index.html

```diff
@@ -38,9 +39,137 @@ <h2 class="intro profession">I'am a Software Developer</h2>
+        <div class="grid-container">
+            <div class="stack">css</div>
+            <div class="stack">html</div>
+            <div class="stack">bootstrap</div>
+            <div class="stack">Ruby</div>
+            <span class="project">See Project</span>
+        </div>
+    </div>
+
+
+        <div class="second">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+        </div>
+
+    <div class="third">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="fourth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="fifth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="sixth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+</section>
+
+<section class="item-3">
+    <div class="about">
+
+    <h1 class="aboutMe">About me</h1>
```

Comment:
- [ ] Kindly use a h2 instead of h1. The h1 tag is just for the title of the website.